tasks/sphinx.py

```diff
@@ -300,6 +300,7 @@ def _parse_columns(self, all_columns_result):
                 'tags': col[6],
                 'suggested_name': col[7],
                 'timespan': col[8],
+                'timespan_sluggified': underscore_slugify('_'.join(col[8])),
```

Comment:
Doesn't this have the same problem than before? This will return all timespans joined by `_`, won't it? I would just do `col[8][0]` to return the default timespan.

Reply:
We are not using `timespan_sluggified` for boundaries, just for numerators, but we are using the `_parse_columns` method for both and, as the boundaries query is returning an `ARRAY_AGG` for timespans, we also need to return an `ARRAY_AGG` for timespans for the numerators and manage both timespans as if they were arrays to avoid the forementioned problems.

Anyway you are right that `col[8][0]` would have the same effect but I think that getting an `ARRAY_AGG` and then keeping only the first element is more confusing than this.